frontend/src/App.vue

```diff
@@ -39,7 +44,18 @@ export default {
   mounted() {
     window.addEventListener('resize', this.onResize)
     this.setRegion()
+    setInterval(() => {
+      this.idleTime++
```

Comment:
Die Idle Time sollte nicht hier, sondern in der Cuppy Componente gemacht werden.

Reply:
Wenn das funktioniert